images/golang-aws/build.yaml

```
@@ -0,0 +1,30 @@
+# Copyright 2021 The Jetstack contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The AWS binaries require a glibc-based distribution, hence using the
+# golang-buster variants
+
+name: golang-aws # Name of the image to be built
+
+variants:
+  "1.16.6":
+    arguments:
+      BASE_IMAGE: "golang:1.16.6-buster"
+      TERRAFORM_VERSION: "1.0.3"
+      BAZEL_VERSION: "4.0.0"
+
+# Image names to be tagged and pushed
+images:
+- ${_REGISTRY}/${_NAME}:${_DATE_STAMP}-${_GIT_REF}-${GO_VERSION}
+- ${_REGISTRY}/${_NAME}:latest-${GO_VERSION}
```

```suggestion
- ${_REGISTRY}/${_NAME}:latest
```

Latest is special cased so docker pull without any tag will still work